map/manage-bookmarks/src/main/java/com/esri/samples/manage_bookmarks/ManageBookmarksSample.java

```diff
@@ -82,14 +83,14 @@ public void start(Stage stage) {
       Label bookmarkLabel = new Label("Bookmarks");
       bookmarkLabel.getStyleClass().add("panel-label");
 
-      // create a list to hold the names of the bookmarks
-      ListView<String> bookmarkNames = new ListView<>();
-      bookmarkNames.setMaxHeight(190);
+      // create a ListView to display the bookmarks
+      ListView<Bookmark> bookmarkListView = new ListView<>();
+      bookmarkListView.setMaxHeight(190);
+      bookmarkListView.setCellFactory(p -> new BookmarkListCell());
 
       // when user clicks on a bookmark change to that location
-      bookmarkNames.getSelectionModel().selectedItemProperty().addListener((ov, old_val, new_val) -> {
-        int index = bookmarkNames.getSelectionModel().getSelectedIndex();
-        mapView.setBookmarkAsync(bookmarkList.get(index));
+      bookmarkListView.getSelectionModel().selectedItemProperty().addListener((ov, old_val, new_val) -> {
+        mapView.setBookmarkAsync(new_val);
```

Comment:
```suggestion
      bookmarkListView.getSelectionModel().selectedItemProperty().addListener((observable, oldValue, newValue) -> {
        mapView.setBookmarkAsync(newValue);
```
In other listener usages we've used this format for these parameters